### PR TITLE
#169564102 backend fix 

### PR DESCRIPTION
**Pull request by nraufu**
> #### What does this PR do?

> fix backend feedback from LFA

> #### Description of Task to be completed

> - remove unused modules
> - fix dev environment
> - fix entry fields validation
> - correcting status codes
> - add info to Readme

> #### What are the relevant pivotal tracker stories?

>#169564102

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "description": "MyDiary is an online journal where users can pen down their thoughts and feelings.",
   "main": "index.js",
   "scripts": {
-    "dev": "node server/index.js --exec babel-node",
+    "dev": "npm run build && node dist/index.js --exec babel-node",
```
Comment (frankhn) [inline package.json]: Is there a reason why you should transpile your project.

**Review by frankhn — CHANGES_REQUESTED** [review]
Nice try, However, have a look at running a node js app both in development and production. I don't think you should be transpiring your project before running it.

File: package.json (hunk 1 above)
Comment (nraufu) [reply package.json]: yes for it to run in node otherwise it won't run

File: package.json [hunk 2]
```diff
@@ -4,9 +4,9 @@
   "description": "MyDiary is an online journal where users can pen down their thoughts and feelings.",
   "main": "index.js",
   "scripts": {
-    "dev": "node server/index.js --exec babel-node",
+    "dev": "nodemon --exec babel-node server/index.js",
     "build": "babel server -d dist",
-    "start": "npm run build && node dist/index.js",
+    "start": "nodemon --exec babel-node server/index.js",
```
Comment (frankhn) [inline package.json]: Nice try, however, are you going to run "nodemon" in production mode?

**Review by frankhn — CHANGES_REQUESTED** [review]
Kindly, review my comment and reach out if you need help.

**Review by frankhn — APPROVED** [review]
LGTM. Good job.

**Pull request merged**